WebRoot/js/zimbraAdmin/accounts/model/ZaFp.js

```diff
@@ -138,10 +138,14 @@ ZaFp.getProviders = function () {
 //@prefix: foreign principal prefixes
 ZaFp.findDupPrefixFp = function (entries, prefix) {
     for (var i =0 ; i < entries.length; i ++) {
-        if (entries[i].indexOf(prefix) == 0) {
+        // if a new provide is ZIMBRA, the prefix is empty string ""
+        if(prefix.length == 0 && entries[i].indexOf(":") == -1) {
+            return true;
+        }
+        if(prefix.length != 0 && entries[i].indexOf(prefix) == 0) {
```

Comment:
Is it intentionally declared two "if" clauses separately?  If not why don't you put two conditions into one "if"?
And put a space between "if" and "(" to follow the Zimbra's coding conventions.

Reply:
Fixed it.